### DISCUSSION TITLE
Why is the mountain shadow / Trrrain always blurred ?

### DISCUSSION THREAD

**Discussion by cnchip** [osmandapp/OsmAnd#16334] [General]
As a comparison, the Locus performs more clearly and comfortably.
Please see my screenshot for comparison at
Why is this? Is it possible to achieve the same effect?
I expect you don't mind questions like this, both of these apps I really like.
Many thanks.
![Snap2](https://user-images.githubusercontent.com/8869425/215024440-52dc538d-7bf7-4efe-89a0-7a8e20b50869.jpg)

**Comment by vshcherb** [comment]
It's not enough resolutions, we're preparing the best resolution imagery this year the biggest challenge is to make computable on device, so it doesn't take too much memory

**Reply by cnchip** [reply]
thanks.  we look forward to further updates.
